frontend/src/metabase/dashboard/components/DashCard/DashCardParameterMapper/DashCardCardParameterMapperContent.tsx

```diff
@@ -103,6 +103,7 @@ export const DashCardCardParameterMapperContent = ({
         heading: t`You can connect widgets to {{variables}} in heading cards.`,
         text: t`You can connect widgets to {{variables}} in text cards.`,
         link: t`You cannot connect variables to link cards.`,
+        iframe: t`You cannot connect variables to iframe cards.`,
```

Comment:
Couldn't we have it work with {{variables}} like text cards? It would unlock a ton of cool stuff.

Reply:
Agree, but we decided to exclude it from the first version but we can add it in future when needed